.buildkite/scripts/promote_release_channel.sh

```diff
@@ -88,15 +88,35 @@ mapfile -t  supervisor_packages < <(echo "${channel_pkgs_json}" | \
                          map(.origin + "/" + .name + "/" + .version + "/" + .release)
                          | .[]')
 
+targets=("x86_64-linux"
```

Comment:
Should we pro-actively add darwin here as well?

Reply:
I'll log a separate issue for darwin so we keep this moving

Reply:
We talked face-to-face about this, but for completeness sake, I'll reply here, too.

We're going to leave the darwin target off for now because we're not yet uploading a macOS hart file to Builder. There's a bit of extra work that'll need to be done to make that happen, but we'll pick this change up when we tackle the rest of them.